src/storage.rs

```diff
@@ -279,7 +294,9 @@ impl Store {
             imports_src: SourceFile::new_empty(IMPORTS_LOCK),
         };
 
-        store.validate(false)?;
+        let today = chrono::NaiveDate::from_ymd_opt(2023, 1, 1).unwrap();
```

Comment:
Yeah, I think there's a solid chance we'll want this to be an argument in the future, I just didn't bother yet, because it'd be a bit of a hassle to change all of the call-sites.